go/chat/utils/utils.go

```diff
@@ -944,6 +948,10 @@ func GetConvMtime(rc types.RemoteConversation) (res gregor1.Time) {
 	conv := rc.Conv
 	var summaries []chat1.MessageSummary
 	for _, typ := range chat1.VisibleChatMessageTypes() {
+		switch typ {
```

Comment:
the description says mtime isn't changing.

Reply:
good point. Removed. The removal means that join/leave bumps convs to the top of the widget, but the snippet shown is still the message prior to the join/leave.